src/infrastructure/congestion.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{
+    net::{NetworkQuality, RttMeasurer},
+    sequence_buffer::{CongestionData, SequenceBuffer},
+    Config,
+};
+
+use std::time::Instant;
+
+/// Type that is responsible for keeping track of congestion information.
+pub struct CongestionHandler {
+    rtt_measurer: RttMeasurer,
+    rtt: f32,
```

Comment:
Probably makes sense to have `rtt` be a method (`rtt()`) where the calculated value of `rtt` lives in the `RttMeasurer`

Reply:
good one!